# Overview

Data sets and software realated to the Microverse cluster. Surveyed by:
- querying Google Scholar for works mentioning the DFG project number 390713860, filtering preprints out, with the Publish or Perish software (exported as (microverse_articles.csv))
- importing the data to Zotero
- retrieving the respective PDFs with Zotero
- searching with pdfgrep for accession numebrs, data availability, unique identifies, and inspecting if the articles have data or software availability sections

# Data

## ArrayExpress

Name|Accession number|Article DOI
---|---|---
RNA-seq of iron-rich pelagic aggregates (iron snow) in 2017 and 2018|[E-MTAB-9686](https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-9686/)|[10.3390/microorganisms9071368](https://doi.org/10.3390/microorganisms9071368)
Response of Candida albicans to Lactobacillus rhamnosus|[E-MTAB-11090](https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-11090/)|[10.1038/s41467-022-30661-5](https://doi.org/10.1038/s41467-022-30661-5)


## BioModels

Name|Model identifier|Article DOI
---|---|---
Ewald2021 - Innate immune response during invasive aspergillosis by dynamic optimization|[MODEL2105110001](https://www.ebi.ac.uk/biomodels/MODEL2105110001)|[10.1371/journal.pcbi.1009645](https://doi.org/10.1371/journal.pcbi.1009645)

## BioProject

Name|Type|Accession number|Bioproject acession number|Article DOI
---|---|---|---|---
Metagenomic shotgun sequencing of marine sediment|Run|SRR8361706|[PRJNA511137](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA511137)|[10.1186/s40168-019-0714-6](https://doi.org/10.1186/s40168-019-0714-6)
Salmonella enterica subsp. enterica serovar Panama strain:18PM0209|BioSample|[SAMN13899429](https://www.ncbi.nlm.nih.gov/biosample/SAMN13899429)|[PRJNA602590](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA602590)|[10.7150/jgen.48656](https://doi.org/10.7150/jgen.48656)
Salmonella enterica subsp. enterica serovar Rissen strain:19CS0416|BioSample|[SAMN13381784](https://www.ncbi.nlm.nih.gov/biosample/SAMN13381784)|[PRJNA591084](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA591084)|[10.7150/jgen.42790](https://doi.org/10.7150/jgen.42790)
Changes to intestinal microbiota after Roux-Y-en Gastric bypass surgery|BioProject|-|[PRJNA561349](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA561349)|[10.1186/s40168-020-0788-1](https://doi.org/10.1186/s40168-020-0788-1)
Human gut metagenome Raw sequence reads upon exercise intervention|BioProject|-|[PRJNA454826](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA454826)|[10.1016/j.cmet.2019.11.001](https://doi.org/10.1016/j.cmet.2019.11.001)
Francisella tularensis subsp. holarctica|BioProject|-|[PRJNA560345](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA560345)|[10.3389/fmicb.2020.00636](https://doi.org/10.3389/fmicb.2020.00636)
Francisella tularensis subsp. holarctica strain:08T0073|BioProject|-|[PRJNA353900](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA353900)|[10.3389/fmicb.2020.00636](https://doi.org/10.3389/fmicb.2020.00636)
Francisella tularensis subsp. holarctica|BioProject|-|[PRJNA575140](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA575140)|[10.3389/fmicb.2020.00636](https://doi.org/10.3389/fmicb.2020.00636)
Clinical and Environmental Aspergillus fumigatus in Germany|BioProject|-|[PRJNA697844](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA697844)|[10.1038/s41564-021-00993-x](https://doi.org/10.1038/s41564-021-00993-x)
Klebsiella grimontii strain:AT013-Mero-001|BioSample|[SAMN17250868](https://www.ncbi.nlm.nih.gov/biosample/SAMN17250868)|[PRJNA690555](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA690555)|[10.1128/MRA.00054-21](https://doi.org/10.1128/MRA.00054-21)
Whole-genome sequencing of the fungal genus Podaxis Genome sequencing and assembly|BioProject|-|[PRJNA681736](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA681736)|[10.1016/j.isci.2021.102680](https://doi.org/10.1016/j.isci.2021.102680)
Arcobacter cibarius and Arcobacter thereius Genome sequencing and assembly|BioProject|-|[PRJNA542783](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA542783)|[10.3389/fcimb.2021.532989](https://doi.org/10.3389/fcimb.2021.532989)
Human gut microbiota Raw sequence reads with and without cancer cachexia|BioProject|-|[PRJNA626477](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA626477)|[10.1371/journal.pone.0259898](https://doi.org/10.1371/journal.pone.0259898)
Gut microbiome is associated with exercise tolerance of resected early-stage lung cancer patients|BioProject|-|[PRJNA668745](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA668745)|[10.1371/journal.pone.0259898](https://doi.org/10.1371/journal.pone.0259898)
Human gut microbiota raw sequences of critically ill patients|BioProject|-|[PRJNA670323](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA670323)|[10.1080/19490976.2021.1993598](https://doi.org/10.1080/19490976.2021.1993598)
Mock communities and Arabidopsis thaliana leaves, Amplification of 16S 18S ITS|BioProject|-|[PRJNA420016](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA420016)|[10.1111/1755-0998.13408](https://doi.org/10.1111/1755-0998.13408)
Development of blocking oligos for avoiding non-target amplification in amplicon sequencing|BioProject|-|[PRJNA663775](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA663775)|[10.1111/1755-0998.13408](https://doi.org/10.1111/1755-0998.13408)
Benchmarking of Short-read- and Hybrid-Assembly to improve the genome analysis in Francisella tularensis|BioProject|-|[PRJNA625652](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA625652)|[10.1186/s12864-021-08115-x](https://doi.org/10.1186/s12864-021-08115-x)
Human vaginal cells interacting with C. glabrata in presense/absense of albumin|BioProject|-|[PRJNA745548](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA745548)|[10.1371/journal.ppat.1010037](https://doi.org/10.1371/journal.ppat.1010037)
Candida glabrata petite genome sequences|BioProject|-|[PRJNA665484](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA665484)|[10.1128/mBio.01128-21](https://doi.org/10.1128/mBio.01128-21)
Clostridium septicum Genome sequencing|BioProject|-|[PRJNA412368](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA412368)|[10.3389/fmicb.2021.771945](https://doi.org/10.3389/fmicb.2021.771945)
Reconstruction of ancient microbial genomes from the human gut|BioProject|-|[PRJNA561510](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA561510)|[10.1038/s41586-021-03532-0](https://doi.org/10.1038/s41586-021-03532-0)
Assessment of total and active bacterial communities in groundwater of two superimposed limestone aquifer assemblages|BioProject|-|[PRJEB14968](https://www.ncbi.nlm.nih.gov/bioproject/PRJEB14968)|[10.1016/j.watres.2021.117290](https://doi.org/10.1016/j.watres.2021.117290)
Bacterial diversity in carbonate rock aquifers of the Hainich Critical Zone Exploratory|BioProject|-|[PRJEB20223](https://www.ncbi.nlm.nih.gov/bioproject/PRJEB20223)|[10.1016/j.watres.2021.117290](https://doi.org/10.1016/j.watres.2021.117290)
Bacterial community structure in soils, seepage water and groundwater of the Hainich Critical Zone Exploratory|BioProject|-|[PRJEB25133](https://www.ncbi.nlm.nih.gov/bioproject/PRJEB25133)|[10.1016/j.watres.2021.117290](https://doi.org/10.1016/j.watres.2021.117290)
Study of endolithic bacteria|BioProject|-|[PRJEB26565](https://www.ncbi.nlm.nih.gov/bioproject/PRJEB26565)|[10.1016/j.watres.2021.117290](https://doi.org/10.1016/j.watres.2021.117290)
Environmental selection shapes the formation of groundwater microbiomes|BioProject|-|[PRJEB33032](https://www.ncbi.nlm.nih.gov/bioproject/PRJEB33032)|[10.1016/j.watres.2021.117290](https://doi.org/10.1016/j.watres.2021.117290)
Composition of bacterial and eukaryotic communities in oligotrophic groundwater of the Hainich Critical Zone Exploratory|BioProject|-|[PRJEB33254](https://www.ncbi.nlm.nih.gov/bioproject/PRJEB33254)|[10.1016/j.watres.2021.117290](https://doi.org/10.1016/j.watres.2021.117290)
Composition and age of dissolved organic matter affect bacterial diversity during groundwater recharge events|BioProject|-|[PRJEB33923](https://www.ncbi.nlm.nih.gov/bioproject/PRJEB33923)|[10.1016/j.watres.2021.117290](https://doi.org/10.1016/j.watres.2021.117290)
Groundwater bacterial communities evolve over time, exhibiting cyclic similarity patterns in response to annual recharge events|BioProject|-|[PRJEB35519](https://www.ncbi.nlm.nih.gov/bioproject/PRJEB35519)|[10.1016/j.watres.2021.117290](https://doi.org/10.1016/j.watres.2021.117290)
Psilocybe mexicana and Psilocybe azurescens genome sequencing and assembly|BioProject|-|[PRJNA779034](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA779034)|[10.1002/cbic.202200249](https://doi.org/10.1002/cbic.202200249)
Human gut microbiota Raw sequence reads from medication-naive individuals who are lean or abdominally obese subjects with and without T2D|BioProject|-|[PRJNA686835](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA686835)|[10.1002/advs.202100536](https://doi.org/10.1002/advs.202100536)
Dark CO2 fixation in temperate forest soils|BioProject|-|[PRJNA607916](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA607916)|[10.5194/bg-2022-90](https://doi.org/10.5194/bg-2022-90)
Gut microbiota of baseline individuals in a 4.6-year non-alcoholic fatty liver disease prospective study|BioProject|-|[PRJNA728908](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA728908)|[10.1126/scitranslmed.abk0855](https://doi.org/10.1126/scitranslmed.abk0855)
Human gut microbiota Raw sequence reads from medication-naive individuals who are lean or abdominally obese subjects with and without T2D|BioProject|-|[PRJNA686835](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA686835)|[10.1126/scitranslmed.abk0855](https://doi.org/10.1126/scitranslmed.abk0855)
ITS sequencing of fungal ribosomal DNA (rDNA) for colonic mucosa lavage samples from non-IBD individuals and patients with UC|BioProject|-|[PRJNA610042](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA610042)|[10.1038/s41586-022-04502-w](https://doi.org/10.1038/s41586-022-04502-w)
Fungal strain-level diversity defines host-mycobiota interactions and inflammation in Inflammatory Bowel Disease|BioProject|-|[PRJNA702809](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA702809)|[10.1038/s41586-022-04502-w](https://doi.org/10.1038/s41586-022-04502-w)
Whole genome sequencing of 182 isolates of the fungal pathogen of humans Candida albicans|BioProject|-|[PRJNA432884](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA432884)|[10.1038/s41586-022-04502-w](https://doi.org/10.1038/s41586-022-04502-w)
Soil metagenome - 18O SIP Nematodes|BioProject|-|[PRJNA662476](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA662476)|[10.1016/j.pedobi.2021.150787](https://doi.org/10.1016/j.pedobi.2021.150787)
Plant Litter Decomposition 2019|BioProject|-|[PRJNA634663](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA634663)|[10.1038/s41598-022-11485-1](https://doi.org/10.1038/s41598-022-11485-1)
Skeletonema costatum strain:RCC75 Genome sequencing and assembly|BioProject|-|[PRJNA647329](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA647329)|[10.1016/j.dib.2022.107931](https://doi.org/10.1016/j.dib.2022.107931)
Groundwater metagenome - H41 Autotrophy SIP|BioProject|-|[PRJNA633367](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA633367)|[10.1038/s41396-021-01163-x](https://doi.org/10.1038/s41396-021-01163-x)
Extremely halophilic and metal tolerant community members from a sulfate-dominated metal-rich environment|BioProject|-|[PRJNA784578](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA784578)|[10.3390/microorganisms10010079](https://doi.org/10.3390/microorganisms10010079)


## BMRB

Name|Dataset DOI|Article DOI
---|---|---
Backbone NMR resonance assignments of the nucleotide binding domain of the Bacillus subtilis ABC multidrug transporter BmrA in the ADP-bound state|[10.13018/BMR51156](https://doi.org/10.13018/BMR51156)|[10.1007/s12104-021-10063-2](https://doi.org/10.1007/s12104-021-10063-2)
Backbone NMR assignments of the human TRPV4 intrinsically disordered N-terminus|[10.13018/BMR51147](https://doi.org/10.13018/BMR51147)|[10.1007/s12104-022-10080-9](https://doi.org/10.1007/s12104-022-10080-9)
Backbone NMR assignments of the chicken TRPV4 intrinsically disordered N-terminus|[10.13018/BMR51172](https://doi.org/10.13018/BMR51172)|[10.1007/s12104-022-10080-9](https://doi.org/10.1007/s12104-022-10080-9)
Backbone NMR assignment of the human TRPV1 ion channel N-terminal intrinsically disordered region|[10.13018/BMR51353](https://doi.org/10.13018/BMR51353)|[10.1007/s12104-022-10093-4](https://doi.org/10.1007/s12104-022-10093-4)
Backbone NMR assignment of the human TRPV2 ion channel N-terminal intrinsically disordered region|[10.13018/BMR51354](https://doi.org/10.13018/BMR51354)|[10.1007/s12104-022-10093-4](https://doi.org/10.1007/s12104-022-10093-4)
Backbone and side-chain NMR assignment of the human TRPV3 ion channel N-terminal intrinsically disordered region|[10.13018/BMR51355](https://doi.org/10.13018/BMR51355)|[10.1007/s12104-022-10093-4](https://doi.org/10.1007/s12104-022-10093-4)


## Dryad

Name|Dataset DOI|Article DOI
---|---|---
Data from: Table-top extreme ultraviolet second harmonic generation|[10.6078/D1Z11P](https://doi.org/10.6078/D1Z11P)|[10.1126/sciadv.abe2265](https://doi.org/10.1126/sciadv.abe2265)

## EDMOND

Name|Dataset DOI|Article DOI
---|---|---
Temperature sensitivity of dark CO2 fixation in temperate forest soils|[10.17617/3.EFHWIY](https://doi.org/10.17617/3.EFHWIY)|[10.5194/bg-2022-90](https://doi.org/10.5194/bg-2022-90)
Multi-omics Characterization of Microbial Metabolic Processes Involved in Plant Litter Decomposition|[10.17617/3.4c](https://doi.org/10.17617/3.4c)|[10.1038/s41598-022-11485-1](https://doi.org/10.1038/s41598-022-11485-1)

## ENA

Name|Study accession|Article DOI
---|---|---
Comparison of extraction methods for recovering ancient microbial DNA from paleofeces|[PRJEB33577](https://www.ebi.ac.uk/ena/browser/view/PRJEB33577)|[10.7717/peerj.9001](https://doi.org/10.7717/peerj.9001)
ArcZ regulation of specialized metabolites|[PRJEB33827](https://www.ebi.ac.uk/ena/browser/view/PRJEB33827)|[10.1038/s41564-020-00797-5](https://doi.org/10.1038/s41564-020-00797-5)
Complete transcriptional profiling of five entomopathogenic Xenorhabdus and Photorhabdus strains|[PRJEB24159](https://www.ebi.ac.uk/ena/browser/view/PRJEB24159)|[10.1038/s41564-020-00797-5](https://doi.org/10.1038/s41564-020-00797-5)
AquaDiva: Whole-metagenome shotgun sequencing of a groundwater ecosystem of the Hainich Critical Zone Exploratory|[PRJEB36505](https://www.ebi.ac.uk/ena/browser/view/PRJEB36505)|[10.1186/s40793-021-00395-w](https://doi.org/10.1186/s40793-021-00395-w)
AquaDiva: Metagenomics assemblies of 32 groundwater samples|[PRJEB36523](https://www.ebi.ac.uk/ena/browser/view/PRJEB36523)|[10.1186/s40793-021-00395-w](https://doi.org/10.1186/s40793-021-00395-w)
16S rRNA ampilcon sequencing of two-year iron rich pelagic aggregates (iron snow) and iron snow microcosms incubated under oxic and anoxic condition|[PRJEB40540](https://www.ebi.ac.uk/ena/browser/view/PRJEB40540)|[10.3390/microorganisms9071368](https://doi.org/10.3390/microorganisms9071368)
The evolution and changing ecology of the African hominid oral microbiome|[PRJEB34569](https://www.ebi.ac.uk/ena/browser/view/PRJEB34569)|[10.1073/pnas.2021655118](https://doi.org/10.1073/pnas.2021655118)
Understanding the microbial biogeography of ancient human dentitions to guide study design and interpretation|[PRJEB46022](https://www.ebi.ac.uk/ena/browser/view/PRJEB46022)|[10.1093/femsmc/xtac006](https://doi.org/10.1093/femsmc/xtac006)
Studying the effect of necromass addition to groundwater samples using stable isotope probing|[PRJEB46919](https://www.ebi.ac.uk/ena/browser/view/PRJEB46919)|[10.1128/spectrum.00437-22](https://doi.org/10.1128/spectrum.00437-22)
Late Neolithic/Early Bronze Age Yersinia pestis genomes|[PRJEB51099](https://www.ebi.ac.uk/ena/browser/view/PRJEB51099)|[10.1073/pnas.2116722119](https://doi.org/10.1073/pnas.2116722119)





## Figshare

Name|Dataset DOI|Article DOI
---|---|---
Obtaining deeper insights into microbiome diversity using a simple method to block host and non-targets in amplicon sequencing - Supplementary Figures 9, 10, 11, 12|[10.6084/m9.figshare.13079597.v2](https://doi.org/10.6084/m9.figshare.13079597.v2)|[10.1111/1755-0998.13408](https://doi.org/10.1111/1755-0998.13408)
Obtaining deeper insights into microbiome diversity using a simple method to block host and non-targets in amplicon sequencing - Figures 3, 4 and 5|[10.6084/m9.figshare.13084094.v3](https://doi.org/10.6084/m9.figshare.13084094.v3)|[10.1111/1755-0998.13408](https://doi.org/10.1111/1755-0998.13408)
Obtaining deeper insights into microbiome diversity using a simple method to block host and non-targets in amplicon sequencing - Figure 2, S2, S3|[10.6084/m9.figshare.13033871.v3](https://doi.org/10.6084/m9.figshare.13033871.v3)|[10.1111/1755-0998.13408](https://doi.org/10.1111/1755-0998.13408)
Obtaining deeper insights into microbiome diversity using a simple method to block host and non-targets in amplicon sequencing - Taxonomy Databases|[10.6084/m9.figshare.13033838.v1](https://doi.org/10.6084/m9.figshare.13033838.v1)|[10.1111/1755-0998.13408](https://doi.org/10.1111/1755-0998.13408)
Lipopeptide-mediated bacterial interaction enables cooperative predator defense|[10.6084/m9.figshare.13585583.v1](https://doi.org/10.6084/m9.figshare.13585583.v1)|[10.1073/pnas.2013759118](https://doi.org/10.1073/pnas.2013759118)

## GenBank

Name|Study accession|Article DOI
---|---|---
Podaxis pistillaris voucher AQ553606 internal transcribed spacer 1, partial sequence; 5.8S ribosomal RNA gene, complete sequence; and internal transcribed spacer 2, partial sequence|[MW430027](https://www.ncbi.nlm.nih.gov/nuccore/MW430027)|[10.1016/j.isci.2021.102680](https://doi.org/10.1016/j.isci.2021.102680)
Podaxis pistillaris voucher AQ553608 internal transcribed spacer 1, partial sequence; 5.8S ribosomal RNA gene and internal transcribed spacer 2, complete sequence; and large subunit ribosomal RNA gene, partial sequence|[MW430028](https://www.ncbi.nlm.nih.gov/nuccore/MW430028)|[10.1016/j.isci.2021.102680](https://doi.org/10.1016/j.isci.2021.102680)
Podaxis pistillaris voucher AQ553610 internal transcribed spacer 1, partial sequence; 5.8S ribosomal RNA gene and internal transcribed spacer 2, complete sequence; and large subunit ribosomal RNA gene, partial sequence|[MW430029](https://www.ncbi.nlm.nih.gov/nuccore/MW430029)|[10.1016/j.isci.2021.102680](https://doi.org/10.1016/j.isci.2021.102680)
Podaxis pistillaris voucher AQ553622 internal transcribed spacer 1, partial sequence; 5.8S ribosomal RNA gene, complete sequence; and internal transcribed spacer 2, partial sequence|[MW430030](https://www.ncbi.nlm.nih.gov/nuccore/MW430030)|[10.1016/j.isci.2021.102680](https://doi.org/10.1016/j.isci.2021.102680)
Podaxis pistillaris voucher AQ554176 internal transcribed spacer 1, partial sequence; 5.8S ribosomal RNA gene, complete sequence; and internal transcribed spacer 2, partial sequence|[MW430031](https://www.ncbi.nlm.nih.gov/nuccore/MW430031)|[10.1016/j.isci.2021.102680](https://doi.org/10.1016/j.isci.2021.102680)
Podaxis pistillaris voucher AQ645840 internal transcribed spacer 1, partial sequence; 5.8S ribosomal RNA gene and internal transcribed spacer 2, complete sequence; and large subunit ribosomal RNA gene, partial sequence|[MW430032](https://www.ncbi.nlm.nih.gov/nuccore/MW430032)|[10.1016/j.isci.2021.102680](https://doi.org/10.1016/j.isci.2021.102680)
Podaxis pistillaris voucher AQ645842 internal transcribed spacer 1, partial sequence; 5.8S ribosomal RNA gene and internal transcribed spacer 2, complete sequence; and large subunit ribosomal RNA gene, partial sequence|[MW430033](https://www.ncbi.nlm.nih.gov/nuccore/MW430033)|[10.1016/j.isci.2021.102680](https://doi.org/10.1016/j.isci.2021.102680)
Podaxis pistillaris voucher AQ645843 small subunit ribosomal RNA gene, partial sequence; internal transcribed spacer 1, 5.8S ribosomal RNA gene, and internal transcribed spacer 2, complete sequence; and large subunit ribosomal RNA gene, partial sequence|[MW430034](https://www.ncbi.nlm.nih.gov/nuccore/MW430034)|[10.1016/j.isci.2021.102680](https://doi.org/10.1016/j.isci.2021.102680)
Podaxis pistillaris voucher AQ742011 internal transcribed spacer 1, partial sequence; 5.8S ribosomal RNA gene, complete sequence; and internal transcribed spacer 2, partial sequence|[MW430035](https://www.ncbi.nlm.nih.gov/nuccore/MW430035)|[10.1016/j.isci.2021.102680](https://doi.org/10.1016/j.isci.2021.102680)
Podaxis pistillaris voucher AQ790499 small subunit ribosomal RNA gene, partial sequence; internal transcribed spacer 1, 5.8S ribosomal RNA gene, and internal transcribed spacer 2, complete sequence; and large subunit ribosomal RNA gene, partial sequence|[MW430036](https://www.ncbi.nlm.nih.gov/nuccore/MW430036)|[10.1016/j.isci.2021.102680](https://doi.org/10.1016/j.isci.2021.102680)
Podaxis pistillaris voucher AQ795752 small subunit ribosomal RNA gene, partial sequence; internal transcribed spacer 1, 5.8S ribosomal RNA gene, and internal transcribed spacer 2, complete sequence; and large subunit ribosomal RNA gene, partial sequence|[MW430037](https://www.ncbi.nlm.nih.gov/nuccore/MW430037)|[10.1016/j.isci.2021.102680](https://doi.org/10.1016/j.isci.2021.102680)
Podaxis pistillaris voucher AQ799166 internal transcribed spacer 1, partial sequence; 5.8S ribosomal RNA gene and internal transcribed spacer 2, complete sequence; and large subunit ribosomal RNA gene, partial sequence|[MW430038](https://www.ncbi.nlm.nih.gov/nuccore/MW430038)|[10.1016/j.isci.2021.102680](https://doi.org/10.1016/j.isci.2021.102680)
Podaxis pistillaris voucher AQ799332 internal transcribed spacer 1, partial sequence; 5.8S ribosomal RNA gene, complete sequence; and internal transcribed spacer 2, partial sequence|[MW430039](https://www.ncbi.nlm.nih.gov/nuccore/MW430039)|[10.1016/j.isci.2021.102680](https://doi.org/10.1016/j.isci.2021.102680)
Podaxis pistillaris voucher AQ799333 internal transcribed spacer 1, partial sequence; 5.8S ribosomal RNA gene and internal transcribed spacer 2, complete sequence; and large subunit ribosomal RNA gene, partial sequence|[MW430040](https://www.ncbi.nlm.nih.gov/nuccore/MW430040)|[10.1016/j.isci.2021.102680](https://doi.org/10.1016/j.isci.2021.102680)
Podaxis pistillaris voucher AQ799334 internal transcribed spacer 1, partial sequence; 5.8S ribosomal RNA gene, complete sequence; and internal transcribed spacer 2, partial sequence|[MW430041](https://www.ncbi.nlm.nih.gov/nuccore/MW430041)|[10.1016/j.isci.2021.102680](https://doi.org/10.1016/j.isci.2021.102680)
Podaxis pistillaris voucher AQ811217 small subunit ribosomal RNA gene, partial sequence; internal transcribed spacer 1, 5.8S ribosomal RNA gene, and internal transcribed spacer 2, complete sequence; and large subunit ribosomal RNA gene, partial sequence|[MW430042](https://www.ncbi.nlm.nih.gov/nuccore/MW430042)|[10.1016/j.isci.2021.102680](https://doi.org/10.1016/j.isci.2021.102680)
Podaxis pistillaris voucher AQ7993333 internal transcribed spacer 1, partial sequence; 5.8S ribosomal RNA gene, complete sequence; and internal transcribed spacer 2, partial sequence|[MW430043](https://www.ncbi.nlm.nih.gov/nuccore/MW430043)|[10.1016/j.isci.2021.102680](https://doi.org/10.1016/j.isci.2021.102680)
Podaxis pistillaris voucher LINN1287.7 small subunit ribosomal RNA gene, partial sequence; internal transcribed spacer 1, 5.8S ribosomal RNA gene, and internal transcribed spacer 2, complete sequence; and large subunit ribosomal RNA gene, partial sequence|[MW430044](https://www.ncbi.nlm.nih.gov/nuccore/MW430044)|[10.1016/j.isci.2021.102680](https://doi.org/10.1016/j.isci.2021.102680)
Podaxis pistillaris voucher LINN1287.8 small subunit ribosomal RNA gene, partial sequence; internal transcribed spacer 1, 5.8S ribosomal RNA gene, and internal transcribed spacer 2, complete sequence; and large subunit ribosomal RNA gene, partial sequence|[MW430045](https://www.ncbi.nlm.nih.gov/nuccore/MW430045)|[10.1016/j.isci.2021.102680](https://doi.org/10.1016/j.isci.2021.102680)
Podaxis pistillaris voucher madm22 internal transcribed spacer 1, partial sequence; 5.8S ribosomal RNA gene, complete sequence; and internal transcribed spacer 2, partial sequence|[MW430046](https://www.ncbi.nlm.nih.gov/nuccore/MW430046)|[10.1016/j.isci.2021.102680](https://doi.org/10.1016/j.isci.2021.102680)
Podaxis pistillaris voucher PREM_57485 internal transcribed spacer 1, partial sequence; 5.8S ribosomal RNA gene and internal transcribed spacer 2, complete sequence; and large subunit ribosomal RNA gene, partial sequence|[MW430047](https://www.ncbi.nlm.nih.gov/nuccore/MW430047)|[10.1016/j.isci.2021.102680](https://doi.org/10.1016/j.isci.2021.102680)
Severe acute respiratory syndrome coronavirus 2 isolate SARS-CoV-2/human/DEU/Jena-vi005159/2020, complete genome|[MW633322](https://www.ncbi.nlm.nih.gov/nuccore/MW633322)|[10.1128/JVI.00110-21](https://doi.org/10.1128/JVI.00110-21)
Severe acute respiratory syndrome coronavirus 2 isolate SARS-CoV-2/human/DEU/Jena-vi005587/2020, complete genome|[MW633323](https://www.ncbi.nlm.nih.gov/nuccore/MW633323)|[10.1128/JVI.00110-21](https://doi.org/10.1128/JVI.00110-21)
Severe acute respiratory syndrome coronavirus 2 isolate SARS-CoV-2/human/DEU/Jena-vi005588/2020, complete genome|[MW633324](https://www.ncbi.nlm.nih.gov/nuccore/MW633324)|[10.1128/JVI.00110-21](https://doi.org/10.1128/JVI.00110-21)
Streptomyces sp. RB110-1, whole genome shotgun sequencing project|[JAEKDS000000000.1](https://www.ncbi.nlm.nih.gov/nuccore/JAEKDS000000000.1)|[10.1128/mSphere.01233-20](https://doi.org/10.1128/mSphere.01233-20)
Janthinobacterium sp. strain AD-342 16S ribosomal RNA gene, partial sequence|[MN547318](https://www.ncbi.nlm.nih.gov/nuccore/MN547318)|[10.1093/femsec/fiab017](https://doi.org/10.1093/femsec/fiab017)
Paracoccus sp. strain AD-343 16S ribosomal RNA gene, partial sequence|[MN547319](https://www.ncbi.nlm.nih.gov/nuccore/MN547319)|[10.1093/femsec/fiab017](https://doi.org/10.1093/femsec/fiab017)
Acinetobacter sp. strain AD-344 16S ribosomal RNA gene, partial sequence|[MN547320](https://www.ncbi.nlm.nih.gov/nuccore/MN547320)|[10.1093/femsec/fiab017](https://doi.org/10.1093/femsec/fiab017)
Flavobacterium sp. strain AD-85 16S ribosomal RNA gene, partial sequence|[MN547321](https://www.ncbi.nlm.nih.gov/nuccore/MN547321)|[10.1093/femsec/fiab017](https://doi.org/10.1093/femsec/fiab017)
Pseudomonas sp. strain AD-122 16S ribosomal RNA gene, partial sequence|[MN547322](https://www.ncbi.nlm.nih.gov/nuccore/MN547322)|[10.1093/femsec/fiab017](https://doi.org/10.1093/femsec/fiab017)
Pseudomonas sp. strain AD-125 16S ribosomal RNA gene, partial sequence|[MN547323](https://www.ncbi.nlm.nih.gov/nuccore/MN547323)|[10.1093/femsec/fiab017](https://doi.org/10.1093/femsec/fiab017)
Pseudomonas sp. SZ57, whole genome shotgun sequencing project|[NZ_WIBD00000000](https://www.ncbi.nlm.nih.gov/nuccore/NZ_WIBD00000000)|[10.1073/pnas.2013759118](https://doi.org/10.1073/pnas.2013759118)
Paenibacillus sp. SZ31, whole genome shotgun sequencing project|[JABBEA000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABBEA000000000)|[10.1073/pnas.2013759118](https://doi.org/10.1073/pnas.2013759118)
Streptomyces mirabilis strain P16B-1 chromosome, complete genome|[CP074102](https://www.ncbi.nlm.nih.gov/nuccore/CP074102)|[10.1128/aem.00085-22](https://doi.org/10.1128/aem.00085-22)
Streptomyces mirabilis strain P16B-1 plasmid pI|[CP074103](https://www.ncbi.nlm.nih.gov/nuccore/CP074103)|[10.1128/aem.00085-22](https://doi.org/10.1128/aem.00085-22)
Skeletonema costatum strain RCC75, whole genome shotgun sequencing project|[JAHBBA000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAHBBA000000000)|[10.1016/j.dib.2022.107931](https://doi.org/10.1016/j.dib.2022.107931)

## GEO

Name|Study accession|Article DOI
---|---|---
Identification of bicyclomycin-sensitive transcripts in Vibrio cholerae|[GSE144478](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE144478)|[10.7554/eLife.58836](https://doi.org/10.7554/eLife.58836)
Global identification of RNase E sites in Vibrio cholerae|[GSE148675](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE148675)|[10.7554/eLife.58836](https://doi.org/10.7554/eLife.58836)
OppZ target identification|[GSE144479](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE144479)|[10.7554/eLife.58836](https://doi.org/10.7554/eLife.58836)
Switching fatty acid metabolism by an RNA-controlled feed forward loop|[GSE140516](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE140516)|[10.1073/pnas.1920753117](https://doi.org/10.1073/pnas.1920753117)
Transcriptomes and Epigenomes of Hemin exposed trained monocytes|[GSE111003](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE111003)|[10.1073/pnas.2102698118](https://doi.org/10.1073/pnas.2102698118)
Innate immune function of galectin-1 as a DAMP|[GSE140892](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE140892)|[10.1038/s41590-020-00844-7](https://doi.org/10.1038/s41590-020-00844-7)
Identification of bicyclomycin-sensitive transcripts in Vibrio cholerae|[GSE168736](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE168736)|[10.15252/embj.2021108542](https://doi.org/10.15252/embj.2021108542)
Study of mating type genes and their roles in mating interactions of the basidiomycete Schizophyllum commune|[GSE26401](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE26401)|[10.3390/jof7050399](https://doi.org/10.3390/jof7050399)
RNA sequencing of early extracellular vesicles derived from human monocytes in presence of C. albicans|[GSE186234](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE186234)|[10.1128/mbio.03563-21](https://doi.org/10.1128/mbio.03563-21)



## HKI Jena Server

Name|Dataset URL|Article DOI
---|---|---
Targeted delivery of a phosphoinositide 3-kinase Î³ inhibitor to restore organ function in sepsis through dye-functionalized lipid nanocarriers|https://asbdata.hki-jena.de/PressEtAl2021_EMBOMolMed|[10.15252/emmm.202114436](https://doi.org/10.15252/emmm.202114436)
Invasive aspergillosis-on-chip: A quantitative treatment study of human Aspergillus fumigatus infection - Raw data|https://asbdata.hki-jena.de/Hoang_Cseresnyes_HartungEtAl2021_Biomaterials/Raw%20data|[10.1016/j.biomaterials.2022.121420](https://doi.org/10.1016/j.biomaterials.2022.121420)
Invasive aspergillosis-on-chip: A quantitative treatment study of human Aspergillus fumigatus infection - Processed data|https://asbdata.hki-jena.de/Hoang_Cseresnyes_HartungEtAl2021_Biomaterials/Processed%20data|[10.1016/j.biomaterials.2022.121420](https://doi.org/10.1016/j.biomaterials.2022.121420)



## MassIVE

Name|Id.|Article DOI
---|---|---
GNPS - Small RNA directs symbiosis, virulence, and natural products biosynthesis in entomopathogenic bacteria|[MSV000084163](https://massive.ucsd.edu/ProteoSAFe/QueryMSV?id=MSV000084163)|[10.1038/s41564-020-00797-5](https://doi.org/10.1038/s41564-020-00797-5)

## Metabolights

Name|Study Identifier|Article DOI
---|---|---
Metabolic profiling with UPLC-MS orbitrap of diatoms Coscinodiscus granii infected with parasitic oomycete Lagenisma coscinodisci|[MTBLS775](https://www.ebi.ac.uk/metabolights/MTBLS775)|[10.1038/s41467-019-12908-w](https://doi.org/10.1038/s41467-019-12908-w)
Metabolomics-derived marker metabolites to characterize Phaeocystis pouchetii in natural plankton communities|[MTBLS758](https://www.ebi.ac.uk/metabolights/MTBLS758)|[10.1038/s41598-020-77169-w](https://doi.org/10.1038/s41598-020-77169-w)
Pronounced uptake and metabolism of organic substrates by diatoms revealed by pulse-labeling metabolomics|[MTBLS2333](https://www.ebi.ac.uk/metabolights/MTBLS2333)|[10.3389/fmars.2022.821167](https://doi.org/10.3389/fmars.2022.821167)

## Metabolomics Workbench

Name|Project DOI|Article DOI
---|---|---
Dual RNA regulator VcdRP in V.cholera modulates central metabolism|[10.21228/M8JD7N](https://10.21228/M8JD7N)|[10.15252/embj.2021108542](https://doi.org/10.15252/embj.2021108542)

## OSF

Name|Id.|Article DOI
---|---|---
AquaDiva: Collection of CPR genomes|[10.17605/OSF.IO/WQ7TR](https://doi.org/10.17605/OSF.IO/WQ7TR)|[10.1186/s40793-021-00395-w](https://doi.org/10.1186/s40793-021-00395-w)
Proteomics Reference database used in Geesink et al (2022): Bacterial necromass is rapidly metabolized by heterotrophic bacteria and supports multiple trophic levels of the groundwater microbiome|[osf.io/59y3w](https://osf.io/59y3w)|[10.1128/spectrum.00437-22](https://doi.org/10.1128/spectrum.00437-22)

## PRIDE

Name|Study Identifier|Article DOI
---|---|---
Intestinal microbiota after RYGB|[PXD013337](https://www.ebi.ac.uk/pride/archive/projects/PXD013337)|[10.1186/s40168-020-0788-1](https://doi.org/10.1186/s40168-020-0788-1)
Small RNA directs symbiosis, virulence, and natural products biosynthesis in entomopathogenic bacteria|[PXD019095](https://www.ebi.ac.uk/pride/archive/projects/PXD019095)|[10.1038/s41564-020-00797-5](https://doi.org/10.1038/s41564-020-00797-5)
Rapid and culture free identification of Francisella in hare carcasses by high-resolution mass spectrometry|[PXD013979](https://www.ebi.ac.uk/pride/archive/projects/PXD013979)|[10.3389/fmicb.2020.00636](https://doi.org/10.3389/fmicb.2020.00636)
Insights into autotrophic activities and carbon flow in iron-rich pelagic aggregates (iron snow)|[PXD025534](https://www.ebi.ac.uk/pride/archive/projects/PXD025534)|[10.3390/microorganisms9071368](https://doi.org/10.3390/microorganisms9071368)
A dual RNA regulator balancing carbon uptake and 1 central metabolism in Vibrio cholerae|[PXD025275](https://www.ebi.ac.uk/pride/archive/projects/PXD025275)|[10.15252/embj.2021108542](https://doi.org/10.15252/embj.2021108542)
Bacterial necromass is rapidly metabolized by heterotrophic bacteria and supports multiple trophic levels of the groundwater microbiome|[PXD031173](https://www.ebi.ac.uk/pride/archive/projects/PXD031173)|[10.1128/spectrum.00437-22](https://doi.org/10.1128/spectrum.00437-22)
Sulfur-fueled chemolithoautotrophs replenish organic carbon inventory in groundwater|[PXD024889](https://www.ebi.ac.uk/pride/archive/projects/PXD024889)|[10.1038/s41396-021-01163-x](https://doi.org/10.1038/s41396-021-01163-x)


## RefSeq

Name|Accession number|Article DOI
---|---|---
Clostridium septicum strain DSM 7534 chromosome, complete genome|[NZ_CP023671.1](https://www.ncbi.nlm.nih.gov/nuccore/NZ_CP023671.1)|[10.3389/fmicb.2021.771945](https://doi.org/10.3389/fmicb.2021.771945)
Clostridium septicum strain DSM 7534 plasmid unnamed1, complete sequence|[NZ_CP023672.1](https://www.ncbi.nlm.nih.gov/nuccore/NZ_CP023672.1)|[10.3389/fmicb.2021.771945](https://doi.org/10.3389/fmicb.2021.771945)

## Zenodo

Name|Dataset DOI|Article DOI
---|---|---
A Versatile and Customizable Low-Cost 3D-Printed Open-Standard for Microscopic Imaging|[10.5281/zenodo.4018965](https://doi.org/10.5281/zenodo.4018965)|[10.1038/s41467-020-19447-9](https://doi.org/10.1038/s41467-020-19447-9)
Cysteinolic acid is a widely distributed compatible solute of marine microalgae|[10.5281/zenodo.5697964](https://doi.org/10.5281/zenodo.5697964)|[10.3390/md19120683](https://doi.org/10.3390/md19120683)
Comparative genomics reveals sanitary and catabolic capabilities of Actinobacteria within the fungus-farming termite symbiosis|[10.5281/zenodo.4302144](https://doi.org/10.5281/zenodo.4302144)|[10.1128/mSphere.01233-20](https://doi.org/10.1128/mSphere.01233-20)

# Software or Database

Name|Article DOI|Archive DOI|Repository URL
---|---|---|---
Sourcepredict|[10.21105/joss.01540](https://doi.org/10.21105/joss.01540)|[10.5281/zenodo.3379604](https://doi.org/10.5281/zenodo.3379604)|https://github.com/maxibor/sourcepredict
nf-core/coproid: coproID v1.0 - Dioptre Walrus|[10.7717/peerj.9001](https://doi.org/10.7717/peerj.9001)|[10.5281/zenodo.2653757](https://doi.org/10.5281/zenodo.2653757)|https://github.com/nf-core/coproid/tree/1.0
UC2 - Open and Modular Optical Toolbox (Hardware)|[10.1038/s41467-020-19447-9](https://doi.org/10.1038/s41467-020-19447-9)|[10.5281/zenodo.4041339](https://doi.org/10.5281/zenodo.4041339)|https://github.com/openUC2/UC2-GIT
UC2 - Open and Modular Optical Toolbox (Software)|[10.1038/s41467-020-19447-9](https://doi.org/10.1038/s41467-020-19447-9)|[10.5281/zenodo.4041343](https://doi.org/10.5281/zenodo.4041343)|https://github.com/openUC2/UC2-Software-GIT
PyDamage|[10.7717/peerj.11845](https://doi.org/10.7717/peerj.11845)|[10.5281/zenodo.4981768](https://doi.org/10.5281/zenodo.4981768)|https://github.com/maxibor/pydamage
ceuMassMediator|[10.3390/jof7050387](https://doi.org/10.3390/jof7050387)|-|https://github.com/albertogilf/ceuMassMediator/tree/master/ceu-mass-mediator-v4.0
CMM Aspergillus DB|[10.3390/jof7050387](https://doi.org/10.3390/jof7050387)|-|https://github.com/albertogilf/ceuMassMediator/tree/master/CMMAspergillusDB
AmpStop|[10.1111/1755-0998.13408](https://doi.org/10.1111/1755-0998.13408)|-|https://github.com/magler1/AmpStop
C. glabrata and human vaginal epithelial cell interactions in presense of human albumin|[10.1371/journal.ppat.1010037](https://doi.org/10.1371/journal.ppat.1010037)|-|https://github.com/Gabaldonlab/C_glabrata_with_albumin
ancient-microbiome-denovo|[10.1038/s41586-021-03532-0](https://doi.org/10.1038/s41586-021-03532-0)|-|https://github.com/kosticlab/ancient-microbiome-denovo
Quantifying the effect of ancient DNA damage on the assembled sequences using simulations|[10.1038/s41586-021-03532-0](https://doi.org/10.1038/s41586-021-03532-0)|-|https://github.com/alexhbnr/effect_aDNAdamage_denovoassembly
AncientMetagenomeDir|[10.1038/s41597-021-00816-y](https://doi.org/10.1038/s41597-021-00816-y)|[10.5281/zenodo.3980833](https://doi.org/10.5281/zenodo.3980833)|https://github.com/SPAAM-community/AncientMetagenomeDir
AncientMetagenomeDirCheck|[10.1038/s41597-021-00816-y](https://doi.org/10.1038/s41597-021-00816-y)|[10.5281/zenodo.4003826](https://doi.org/10.5281/zenodo.4003826)|https://github.com/SPAAM-community/AMDirT
Evolution of the Hominid Calculus Microbiome|[10.1073/pnas.2021655118](https://doi.org/10.1073/pnas.2021655118)|[10.5281/zenodo.3740493](https://doi.org/10.5281/zenodo.3740493)|https://github.com/jfy133/Hominid_Calculus_Microbiome_Evolution
JIPipe|[10.21203/rs.3.rs-1641739/v1](https://doi.org/10.21203/rs.3.rs-1641739/v1)|[10.5281/zenodo.6532719](https://doi.org/10.5281/zenodo.6532719)|https://github.com/applied-systems-biology/JIPipe/
Calbicans LRhamnosus Epithelial Cells Interaction|[10.1038/s41467-022-30661-5](https://doi.org/10.1038/s41467-022-30661-5)|[10.5281/zenodo.6501838](https://doi.org/10.5281/zenodo.6501838)|https://github.com/mohammadmirhakkak/Calbicans_LRhamnosus_EpithelialCells_Interaction
sam2lca: Lowest Common Ancestor for SAM/BAM/CRAM alignment files|[10.21105/joss.04360](https://doi.org/10.21105/joss.04360)|[10.5281/zenodo.6585307](https://doi.org/10.5281/zenodo.6585307)|https://github.com/maxibor/sam2lca
Scripts used for analyses in the project: "Understanding the microbial biogeography of ancient human dentitions to guide study design and interpretation"|[10.1093/femsmc/xtac006](https://doi.org/10.1093/femsmc/xtac006)|[10.5281/zenodo.6034700](https://doi.org/10.5281/zenodo.6034700)|https://github.com/ZandraFagernas/dental_arcade
The code deposited here contains the genome assembly workflow as described in the manuscript "Draft genome assembly of the marine diatom Skeletonema costatum RCC75 " by Sorokina et al.|[10.1016/j.dib.2022.107931](https://doi.org/10.1016/j.dib.2022.107931)|[10.5281/zenodo.5862116](https://doi.org/10.5281/zenodo.5862116)|https://github.com/Bioinformatics-Core-Facility-Jena/SE20200226_16
LNBAplague|[10.1073/pnas.2116722119](https://doi.org/10.1073/pnas.2116722119)|-|https://github.com/aidaanva/LNBAplague
